src/applications/vaos/actions/expressCare.js

```diff
@@ -188,31 +191,36 @@ export function submitExpressCareRequest(router) {
   return async (dispatch, getState) => {
     const expressCare = getState().expressCare;
     const formData = expressCare.newRequest.data;
+    let activeFacility;
+    let additionalEventData = {};
 
-    const activeFacility = selectActiveExpressCareFacility(
-      getState(),
-      moment.utc(),
-    );
+    try {
```

Comment:
I moved the try block up because I was getting an error when setting additional event data and it was hard to track down because it was outside the try/catch.